bin/staging_area_monitor

```diff
@@ -32,32 +39,49 @@ my @area_list = $monitor->validate_areas(@ARGV);
 
 croak 'No valid arguments' if scalar @area_list == 0;
 
-my $iteration = 0;
+# In production dev variable is not set
+my $dry_run = $ENV{'dev'}; 
```

Comment:
> Intentional loss of
> |$ENV{dev} and ( $ENV{dev} ne 'live' )|
> logic?
>
Yes. This dry_run facility is, unfortunately, not clean, ie some tags 
are still set for the run. Therefore, if one wants to test the whole 
script, either more work has to be done or dev=dev still should be used. 
I was using it with dev=live and an early return so most of the code is 
not executed, but I still get true representation of the status of the 
file system in conjunction with the run status. Can put some extra time 
into setting tags conditionally, then we'll have a true dry_run mode




-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.